life_expectancy/main.py

```diff
@@ -0,0 +1,38 @@
+"""Module to clean dataframe"""
+import sys
+
+import pandas as pd
+
+from life_expectancy.cleaning import JSONCleaner, CSVCleaner
+from life_expectancy.data_access import save_data
+
+from life_expectancy.country import Country
+from life_expectancy.file_type import FileExtension
+
+
+def main(filename: str = None, file_type: FileExtension = FileExtension.CSV) -> pd.DataFrame:
```

Comment:
PEP8 states that lines shall not exceed 79 'cols'

Reply:
@fddelgadoN, are you following pop8?